Edit Introduction Chapter (#114)
tidy-survey-r/tidy-survey-book: 01-introduction.Rmd

# Introduction {#c01-intro}

Surveys are valuable tools for gathering information about a population, used by researchers, governments, and businesses alike to better understand public opinion and behaviors. For example, a non-profit group may analyze societal trends to measure their impact, government agencies may study behaviors to inform policy, or companies may seek to learn customer product preferences to refine business strategy. With survey data, we can explore the world around us. <!-- thinking how we can tie 'exploring' in here -->

> This book focuses on how we **analyze** the data collected from a complex survey design. Complex surveys use weights to adjust for unequal probabilities of selection, non-response, and post-stratification. These adjustments ensure the sample accurately represents the population of interest [@gard2023weightsdef]. To account for the intricate nature of the survey design, analysts rely on statistical software such as SAS, Stata, SUDAAN, and R. 

Reviewer: ```suggestion
Surveys are often conducted with a sample of the population. Therefore, in order to use the survey data to understand the population, we use weights to adjust the survey results for unequal probabilities of selection, non-response, and post-stratification. These adjustments ensure the sample accurately represents the population of interest [@gard2023weightsdef]. To account for the intricate nature of the survey design, analysts rely on statistical software such as SAS, Stata, SUDAAN, and R. 
```

> In this book, we focus on R to introduce survey analysis. Our goal is to provide a comprehensive guide for individuals new to survey analysis but with some familiarity with statistics and R programming. We use a combination of the {survey} and {srvyr} packages and present the code following best practices from the tidyverse. <!-- feel that this paragraph skips around a bit but tried to tie it together-->

Reviewer: ```suggestion
In this book, we focus on R to introduce survey analysis. Our goal is to provide a comprehensive guide for individuals new to survey analysis but with some familiarity with statistics and R programming. We use a combination of the {survey} and {srvyr} packages and present the code following best practices from the tidyverse and assume weights have already been calculated and are available. 
```

## Survey analysis in R

> The {survey} package was released on the Comprehensive R Archive Network (CRAN) in 2003 and has been continuously developed over time^[https://cran.r-project.org/src/contrib/Archive/survey/]. This package, primarily authored by Thomas Lumley, offers an extensive array of features, including:

Reviewer: ```suggestion
The {survey} package was released on the [Comprehensive R Archive Network (CRAN)](https://cran.r-project.org/src/contrib/Archive/survey/) in 2003 and has been continuously developed over time. This package, primarily authored by Thomas Lumley, offers an extensive array of features, including:
```

* Calculation of point estimates and their associated variances, including means, totals, ratios, quantiles, and proportions
* Estimation of regression models, including generalized linear models, log-linear models, and survival curves
* Variances by Taylor linearization or by replicate weights, including balance repeated replication, jackknife, bootstrap, multistage bootstrap, or user-supplied methods
* Hypothesis testing for means, proportions, and other parameters

The {srvyr} package builds on the {survey} package by providing wrappers for functions that align with the tidyverse philosophy. This is our motivation for using and recommending this package. We find that the {srvyr} package is user-friendly for those familiar with the tidyverse packages in R.

> For example, while many functions in the {survey} package use variables as formulas, the {srvyr} package uses tidy selection to pass variable names^[https://dplyr.tidyverse.org/reference/dplyr_tidy_select.html], a common feature in the tidyverse. Users of the tidyverse are likely familiar with the magrittr pipe operator (`%>%`), which seamlessly works with functions from the {srvyr} package. Moreover, several common functions from {dplyr}, such as `filter()`, `mutate()`, and `summarize()`, can be applied to survey objects. This enables users to streamline their analysis workflow and leverage the benefits of both the {srvyr} and tidyverse packages.

Reviewer: ```suggestion
For example, while many functions in the {survey} package use variables as formulas, the {srvyr} package uses [tidy selection](https://dplyr.tidyverse.org/reference/dplyr_tidy_select.html) to pass variable names, a common feature in the tidyverse. Users of the tidyverse are likely familiar with the magrittr pipe operator (`%>%`), which seamlessly works with functions from the {srvyr} package. Moreover, several common functions from {dplyr}, such as `filter()`, `mutate()`, and `summarize()`, can be applied to survey objects. This enables users to streamline their analysis workflow and leverage the benefits of both the {srvyr} and tidyverse packages.
```

  - Statistical testing methods
  - Comparison of means and proportions
  - Goodness of fit tests, tests of independence, and tests of homogeneity
- **Chapter \@ref(c07-modeling) - Modeling**:
  - Linear regression, ANOVA, and logistic regression modeling
- **Chapter \@ref(c08-communicating-results) - Communicating results**:
  - Strategies for communicating survey results
  - Tools and guidance for creating publishable tables and graphs
- **Chapter \@ref(c09-reprex-data) - Reproducible research**: 
  - Various tools and methods for achieving reproducibility
- **Chapter \@ref(c10-specifying-sample-designs) - Sample designs and replicate weights**: 
  - Description of common sampling designs and how to specify in R
  - Description of replicate weight methods and how to specify in R
- **Chapter \@ref(c11-missing-data) - Missing data**:
  - Overview of missing data in surveys
  - Approaches to dealing with missing data
- **Chapter \@ref(c12-recommendations) - Successful survey analysis recommendations**:
  - Tips for successful analysis
  - Debugging skills
- **Chapter \@ref(c13-ncvs-vignette) - National Crime Victimization Survey Vignette**: 

>   - Vignette on analyzing National Crime Victimization Survey (NCVS) data.

Reviewer: ```suggestion
  - Vignette on analyzing National Crime Victimization Survey (NCVS) data
```

  - Illustrates analysis requiring multiple files for victimization rates
- **Chapter \@ref(c14-ambarom-vignette) - AmericasBarometer Vignette**:
  - Vignette on analyzing AmericasBarometer survey data
  - Includes making choropleth maps with survey estimates

The majority of chapters contain code that you can follow. Each of these chapters starts with a "set-up" section, which includes the code needed to load the packages and datasets. We then provide the main idea of the chapter and examples of how to use the functions. Most chapters conclude with exercises to work through. We provide the solutions to the exercises in the online version of the book, available at [tidy-survey-r.github.io](https://tidy-survey-r.github.io/).

While we provide a brief overview of survey methodology and statistical theory, this book is not intended to be the sole resource for these topics. We reference other materials throughout the book and encourage readers to seek those out for more information. 

## Prerequisites

> We assume that you have already conducted a survey and have the data or obtained a microdata file. Microdata, also known as respondent-level or row-level data, differs from summarized data typically found in tables. It contains individual survey responses, along with analysis weights and design variables such as strata or clusters.

Reviewer: ```suggestion
To get the most of our this book, we assume that you have already conducted a survey and have the data or obtained a microdata file. Microdata, also known as respondent-level or row-level data, differs from summarized data typically found in tables. It contains individual survey responses, along with analysis weights and design variables such as strata or clusters.
```

> For the purposes of this book, you need the weights and design variables for your survey data. These are required to accurately calculate unbiased estimates^[If you do not already have weights created for the survey data you are using, we recommend reviewing other resources focused on weight creation such as @Valliant2018weights]. The concepts and techniques discussed in this book will help you to extract meaningful insights from your survey data.

Reviewer: ```suggestion
Additionally, the survey data should already include weights and design variables. These are required to accurately calculate unbiased estimates. The concepts and techniques discussed in this book will help you to extract meaningful insights from your survey data, but will not cover how to create weights in the first place as this is a separate complex topic. If you do not already have weights created for the survey data you are using, we recommend reviewing other resources focused on weight creation such as @Valliant2018weights.
```

* Install and load packages from CRAN and GitHub repositories
* Run R code
* Read data from a folder or their working directory
* Understand fundamental tidyverse concepts such as tidy/long/wide data, tibbles, the magrittr pipe (`%>%`), and tidy selection
* Use the tidyverse packages to wrangle, tidy, and visualize data

If these concepts or skills are new to you, we recommend starting with introductory resources to cover these topics before reading this book. R for Data Science [@wickham2023r4ds] is a beginner-friendly guide for getting started in data science using R. It offers guidance on preliminary installation steps and basic R syntax, and it introduces tidyverse concepts and packages.

## Datasets used in this book

We work with two key datasets throughout the book: the Residential Energy Consumption Survey [RECS -- @recs-2020-tech] and the American National Election Studies [ANES -- @debell]. We introduce and demonstrate the loading and preparation of these datasets in Chapter \@ref(c04-getting-started).

## Conventions

Throughout the book, we use the following typographical conventions:

* Package names are surrounded by curly brackets: {srvyr}

> * Function names are in `constant width` and then parentheses: `survey_mean()`

Reviewer: ```suggestion
* Function names are in constant width text format and include parentheses: `survey_mean()`
```

> * Object and variable names are in constant width: `anes_des`

Reviewer: ```suggestion
* Object and variable names are in constant width text format: `anes_des`
```

## Getting help

We recommend first trying to resolve errors and issues independently using the tips provided in **Chapter \@ref(c12-recommendations)**. 

> If you have questions or face issues while working through the book, please report them in its [GitHub repository](https://github.com/tidy-survey-r/tidy-survey-book).

Reviewer: ```suggestion
If you have questions or face issues while working through the book, please report them to its [GitHub repository](https://github.com/tidy-survey-r/tidy-survey-book).
```

> There are several community forums for asking questions:

Reviewer: ```suggestion
There are several community forums for asking questions, including:
```
